cashctrl_ledger/tests/test_accounts.py

```diff
@@ -14,165 +14,139 @@
+
+    TAX_CODES = BaseTestAccounts.TAX_CODES.copy()
+    # In CashCtrl it is not possible to create TAX CODE without specified account
+    TAX_CODES = TAX_CODES[~(TAX_CODES["id"] == "EXEMPT")]
+
     @pytest.fixture()
-    def ledger(self, initial_ledger):
-        initial_ledger.clear()
-        return initial_ledger
-
-    @pytest.fixture(scope="class")
-    def ledger_with_balance(self, initial_ledger):
-        initial_ledger.restore(accounts=self.ACCOUNTS, ledger=LEDGER_ENTRIES, settings=self.SETTINGS)
-        return initial_ledger
-
-    # TODO: Move this test to the PyLedger package when storing Price history implemented
-    @pytest.mark.parametrize(
-        "account, date, expected",
-        [
-            (9991, "2024-02-21", {"USD": 0.00, "reporting_currency": 0.00}),
-            (9991, "2024-02-22", {"USD": 100.00, "reporting_currency": 88.88}),
-            (9991, "2024-03-23", {"USD": 200.00, "reporting_currency": 174.43}),
-            (9991, "2024-04-24", {"USD": 100.00, "reporting_currency": 96.66}),
-            (9991, "2024-06-26", {"USD": 0.00, "reporting_currency": 1.11}),
-            (9991, None, {"USD": 0.00, "reporting_currency": 1.11}),
-            (9992, "2024-01-20", {"CHF": 0.00, "reporting_currency": 0.00}),
-            (9992, "2024-01-21", {"CHF": 100.00, "reporting_currency": 100.00}),
-            (9992, "2024-05-25", {"CHF": 110.00, "reporting_currency": 110.00}),
-            (9992, None, {"CHF": 110.00, "reporting_currency": 110.00}),
-            (9994, "2024-02-21", {"USD": 0.00, "reporting_currency": 0.00}),
-            (9994, "2024-02-22", {"USD": -100.00, "reporting_currency": -88.88}),
-            (9994, "2024-04-24", {"USD": 0.00, "reporting_currency": -11.11}),
-            (9994, None, {"USD": 0.00, "reporting_currency": -11.11}),
-            (9995, "2024-01-20", {"CHF": 0.00, "reporting_currency": 0.00}),
-            (9995, "2024-01-21", {"CHF": -100.00, "reporting_currency": -100.00}),
-            (9995, "2024-03-23", {"CHF": -185.55, "reporting_currency": -185.55}),
-            (9995, "2024-05-25", {"CHF": -195.55, "reporting_currency": -195.55}),
-            (9995, "2024-06-26", {"CHF": -100.00, "reporting_currency": -100.00}),
-            (9995, None, {"CHF": -100.00, "reporting_currency": -100.00}),
-        ],
-    )
-    def test_account_single_balance(self, ledger_with_balance, account, date, expected):
-        balance = ledger_with_balance._single_account_balance(account=account, date=date)
-        assert balance == expected
-
-    def test_add_already_existed_raise_error(self, ledger):
-        super().test_add_already_existed_raise_error(
-            ledger, error_class=RequestException, error_message="This number is already used"
+    def engine(self, initial_engine):
+        initial_engine.clear()
+        return initial_engine
+
+    def test_account_accessor_mutators(self, engine):
+        super().test_account_accessor_mutators(engine, ignore_row_order=True)
```

Comment:
Here's a possibility to further abstract `ignore_row_order` (outside of the scope of the present PR):
- Create a property `ignore_row_order` that is set in the initiator (defaults to False).
- Use this property throughout the single test methods.

Benefit: Avoids overriding multiple method just to override the same argument every time.